Tests/Marketplace/Tests/search_and_install_packs_test.py

```diff
@@ -133,6 +123,7 @@ def test_search_and_install_packs_and_their_dependencies(mocker):
 
     client = MockClient()
 
+    mocker.patch.object(script, 'install_packs')
```

Comment:
how is this related to your changes?

Reply:
This test was throwing errors in the console for no reason. It should've been done when this was written.